features/service-operation.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Command output from service operations like stop/start/restart
+  service-manager should return the correct exit code from stop/start/restart command
+
+  @operation
+  Scenario Outline: Boot and execute service operations like stop/start/restart
+    Given box is <box>
+    And provider is <provider>
+    And a file named "Vagrantfile" with:
+    """
+    require 'vagrant-libvirt'
+
+    Vagrant.configure('2') do |config|
+      config.vm.box = '<box>'
+      config.vm.box_url = 'file://../boxes/<box>-<provider>.box'
+      config.vm.network :private_network, ip: '<ip>'
+      config.vm.synced_folder '.', '/vagrant', disabled: true
+      config.servicemanager.services = 'docker'
+    end
+    """
+
+    When I successfully run `bundle exec vagrant up --provider <provider>`
+      And I successfully run `bundle exec vagrant service-manager stop docker`
```

Comment:
Sure, just running against docker is fine. But you still should verify that the bahavior is correct.
Regarding OpenShift, maybe only test start/stop/restart against the CDK, there OpenShift is enabled per default. On ADB it OpenShift testing is indeed a bit of a pita, since it needs to pull the image. We should find a way to do it though. One could write the test, but only run it on demand, using an annotation.